### DISCUSSION TITLE
Cloudflare security check required. Please visit chat.openai.com once and come back.

### DISCUSSION THREAD

**Discussion by jiangxin1112** [GorvGoyl/ChatGPT-Writer#54] [Report Bugs and Issues]
I've updated Chrome and re-installed the extension. I still get this error: Cloudflare security check required. Please visit chat.openai.com once and come back. i can't use it !! How to solve it？please report it here.

**Comment by RezaKashizadeh** [comment]
same here

**Reply by dengyunpu** [reply]
i got the same problem, can not work

**Reply by aprazer** [reply]
I can never get cloudflare to let me sign in for join so can't use the program.

**Reply by xhzhou76** [reply]
Same issue.

**Reply by Joechiux** [reply]
me too.

**Reply by cnnggg** [reply]
same

**Reply by ywanglab** [reply]
same issue. Wonder if this is intended. They mention you need to sign up for 14.99/month to ged rid of the problems. about to get rid of the extension.

**Reply by GorvGoyl** [reply]
the issue is from openai chatgpt site so i don't have any control over that. some users see less cloudflare check errors if they keep that  chatgpt tab open

**Reply by ChrisGreen1963** [reply]
I've just signed up to Cloudflare Free option and verified

Didn't set anything in account

Seems to have resolved the problem

Am on free version of chatgpt

**Comment by sauli98** [comment]
after hit the generate button there is no response? how to solve it?

**Comment by echoi0828** [comment]
try using Firefox instead of Chrome.  It worked for me.

**Comment by kpdixith** [comment]
I am also facing this same problem from last 2 days. Pls let me know how do i fix this issue.
